### DISCUSSION TITLE
How to validate mongoDB id?

### DISCUSSION THREAD

**Discussion by MunirBash** [jenssegers/laravel-mongodb#2484] [Q&A]
Im trying to validate the mongoDB id (coming in from API request) but I cant find a function from this library to do the ID validation. 
I tried using this https://www.php.net/manual/en/mongodb-bson-objectid.tostring.php but latest Laravel Sails doesnt seem to have this extension.

**Comment by jinrenjie** [comment]
@MunirBash There is no need to use ObjectID to verify that the ID is valid，You can use validation rules like alpha|size:24
